src/dex/swell/swell.ts

```diff
@@ -123,9 +155,13 @@ export class Swell
     side: SwapSide,
     blockNumber: number,
   ): Promise<string[]> {
-    return this.isEligibleSwap(srcToken, destToken, side)
-      ? [this.getPoolIdentifierKey()]
-      : [];
+    if (!this.isEligibleSwap(srcToken, destToken, side)) return [];
+    if (destToken.address === this.swETHAddress) {
+      return [this.getPoolIdentifierKeySwETH()];
+    } else if (destToken.address === this.rswETHAddress) {
+      return [this.getPoolIdentifierKeyRswETH()];
+    }
+    return [];
```

Comment:
This line can't get triggered?

Reply:
Latest commit removes this unreachable code